README.md

```diff
@@ -16,11 +16,9 @@ Add the plugin to your Webpack configuration:
 
   resolve: {
     plugins: [new BowerResolvePlugin()],
-    modules: [
-        'bower_components'
-    ],
-    descriptionFiles: ['bower.json', 'package.json'],
-    mainFields: ['main', 'browser']
+    modules: ['bower_components', 'node_modules'],
+    descriptionFiles: ['.bower.json', 'package.json'],
```

Comment:
Well, many packages can be installed as a bower dependancy even if they don't have bower.json . I do understand your point. Do you have any ideas on how to report missing descriptor and passback to a resolver chain to look for other descriptors if the one found doesn't have "main" attribute? It could be done in a hacky way but I would prefer to use as much ``enhanced-resolve`` plugin's funcionality as possible.

The main motivation for this is on how ``enhanced-resolve`` plugin works, as it doesn't care if it's bower or npm package, it just looks for descriptor files (in set directories) and tries to match files in defined attributes for resolve.